### ISSUE TITLE
Searching with ace-jump-mode, moves the mark to the first matching char

### ISSUE THREAD

**Issue by ferfebles** [tam17aki/ace-isearch#11]
Usually, when you search something, the mark should be where the point was.

But when I search only for one character (using ace-jump-mode) the point moves to the first matching character, and then, when I press the 'red character' that ace-jump-mode shows, the mark ends in the wrong position: not where I started searching, but on the first matching character.

It can be seen clearly if you use visible-mark while searching.

**Comment by tam17aki** [comment]
Hi,

Thank you for using ace-isearch.

Will the following code solve the problem ... ?
Anyway, I will consider it.

```lisp:
(add-hook 'ace-jump-mode-end-hook #'(lambda () (pop-mark)))
```

**Comment by ferfebles** [comment]
Solved!!

Thanks for your work!

**Comment by ferfebles** [comment]
Reviewing some error messages in emacs, I've found the real problem. I'm using visible-mark.el and pasted some configuration code without reading it ... :-(

```elisp
(setq visible-mark-max 2)
(setq visible-mark-faces `(visible-mark-face1 my-visible-mark-face2))
```

The problem was that **my** word before visible-mark-face2. I didn't had that variable defined.

When I solved that, ace-isearch worked without needing the ace-jump-mode-end-hook patch.

Again, thanks for your code!
